pkg/types/tuf/tuf.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright © 2021 Bob Callaway <bcallawa@redhat.com>
```

Comment:
can you change this to be consistent with `COPYRIGHT.txt`? Just don't want to take credit for work I didn't do :)

Reply:
Hah, I didn't notice!! Sure. :)

Reply:
That was my bad for copying @bobcallaway's homework...